release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,15 @@
+## Version 2.3.0.0 2022-09-09
+
+Compatible with OpenSearch 2.3.0
+
+### Enhancement 
+* Change alignment of Snapshot Management panels in pages/Main/Main.tsx ([#236](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/236))
+* Update Rollup help text ([#221](https://github.com/opensearch-project/index-management-dashboards-plugin/pull/221))
```

Comment:
I believe this text update was already mentioned in these release notes https://github.com/opensearch-project/index-management-dashboards-plugin/pull/222. Looks like PR 221 was just a backport.

Reply:
Fixing...